OpenMTR/Code/Odometer.cs

```diff
@@ -20,37 +20,42 @@ public class Odometer
             { "9", new int[] {1,1,1,1,0,1,1 } }
         };
 
-        public static string Read(Mat sourceImage)
+        public static string Read(Meter meter, List<Rect> digits)
         {
-            return ReadDigits(sourceImage, ExtractDigits(sourceImage));
+            return ReadDigits(meter, SortDigits(digits));
         }
 
-        private static List<Rect> ExtractDigits(Mat image)
+        private static string ReadDigits(Meter meter, List<Rect> digits)
         {
-            List<Rect> digits = new List<Rect>();
-
-            ImageUtils.ColorToGray(image, image);
-            Cv2.GaussianBlur(image, image, new Size(5, 5), 0);
-            Cv2.AdaptiveThreshold(image, image, 250, AdaptiveThresholdTypes.GaussianC, ThresholdTypes.Binary, 5, -1.2);
-            Cv2.MorphologyEx(image, image, MorphTypes.Open, ImageUtils.GetKernel(new Size(3,3)));
-            Cv2.FindContours(image, out Point[][] contours, out HierarchyIndex[] hierarchy, RetrievalModes.List, ContourApproximationModes.ApproxSimple);
-
-            foreach (Point[] point in contours)
+            string digitRead = "";
+            int count = 0;
```

Comment:
Is the count variable necessary to keep here? I see where it is incremented but we don't do anything with it.

Reply:
Debug code, I forgot to remove.